README.md

```diff
@@ -21,7 +21,7 @@ xmlns:chart="using:Syncfusion.UI.Xaml.Charts"
 using Syncfusion.UI.Xaml.Charts;
 ```
 ### Step 3: 
-Initialize an empty chart with XAxes and YAxes as shown in the following code sample.
+Initialize an empty chart with the XAxes and YAxes as shown in the following code sample.
```

Comment:
@DevakumarD : Attach API reference links for APIs.

Reply:
@MuneeshKumarG updated the API reference link.